p2-repository/org.eclipse.collections/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
```

Comment:
I'll add the following text two the README:

Note, the `p2-repository` POM acts as a parent for recipes and also integrates the
recipes into the build process as modules. Each recipe is a Maven project with `eclipse-bundle-recipe` type packaging.